Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -268,7 +292,7 @@ void ComputeEdgeInsets(UIButton button, Button.ButtonContentLayout layout)
 			{
 				button.ImageEdgeInsets = new UIEdgeInsets(0, labelWidth + spacing, 0, -labelWidth - spacing);
 				button.TitleEdgeInsets = new UIEdgeInsets(0, -imageWidth - spacing, 0, imageWidth + spacing);
-				button.ContentEdgeInsets = new UIEdgeInsets(0, 2 * spacing, 0, 2 * spacing);
+				UpdateContentEdge (button, new Thickness (0, 2 * spacing, 0, 2 * spacing));
```

Comment:
Thickness is Left, top, right, bottom
UIEdgeInset is Top, right, bottom, left

So I think all these places where you replaced UIEdgeInsets with Thickness will need to account for this